src/xcode/ENA/ENA/Source/AppDelegate & Globals/__tests__/ENAUITestsQuickActions.swift

```diff
@@ -0,0 +1,147 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+import ExposureNotification
+
+class ENAUITestsQuickActions: XCTestCase {
+
+
+	private let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
+	private var cwaBundleDisplayName = "Corona-Warn" // dynamic app name!
```

Comment:
I don't understand the comment "// dynamic app name!"

Reply:
There was a "TODO:" prefixed previously. It's just a reminder to use the `BundleDisplayName` instead but honestly, we don't change that in this Project so I used the simple solution rather than digging more holes the build & test process might fall into.

Reply:
It's fine for me, but please either change the comment, so its clear and understandable, or remove it.

Reply:
I found a solution that should be fine for everybody